dotCMS/src/main/java/com/dotmarketing/servlets/ShortyServlet.java

```diff
@@ -74,6 +80,20 @@ protected void service(final HttpServletRequest request, final HttpServletRespon
     }
   }
 
+  
+    private Optional<String> getParameter(final String uri, final String paramName) {
+
+        int start  = uri.indexOf("/" + paramName + "/") + paramName.length() + 2;
+        int end  = uri.indexOf("/", start) >-1 ? uri.indexOf("/", start) : uri.length();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'end' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)